scapy/arch/windows/__init__.py

```diff
@@ -83,55 +83,89 @@ def _windows_title(title=None):
         title = title or "Scapy v" + conf.version
         ctypes.windll.kernel32.SetConsoleTitleW(title)
 
-def _suppress_file_handles_inheritance(r=100):
+def _suppress_file_handles_inheritance(r=1000):
     """HACK: python 2.7 file descriptors.
 
     This magic hack fixes https://bugs.python.org/issue19575
     by suppressing the HANDLE_FLAG_INHERIT flag to a range of
     already opened file descriptors.
     """
-    # See https://github.com/secdev/scapy/issues/1136
+    # Fix https://bugs.python.org/issue19575
+    # and https://github.com/secdev/scapy/issues/1136
+    # Bug was fixed on python 3.4+
```

Comment:
Could you remove the duplicated line (see above), and merge this with the docstring ?